packages/validation/src/types.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type the yup schema shape
```

Comment:
please use just `types.ts` this usage of `d.ts` is incorrect > https://stackoverflow.com/questions/29196657/what-is-the-difference-between-d-ts-vs-ts-in-typescript

Reply:
done